implementations/zarrita/environment.yml

```diff
@@ -0,0 +1,10 @@
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python
+  - scikit-image
+  - numcodecs
+  - pip
+  - pip:
+      - git+https://github.com/grlee77/zarrita.git@codec_update
```

Comment:
Not a blocker, but maybe we should follow up on getting PR ( https://github.com/alimanfoo/zarrita/pull/36 ) in?

Reply:
Maybe next step is to move to @normanrz's implementation which has been updated?

Reply:
See https://github.com/joshmoore/zarr_implementations/pull/3